src/main/java/seedu/module/model/task/Workload.java

```diff
@@ -75,6 +79,11 @@ public boolean equals(Object other) {
                 && workloadLevel.equals(((Workload) other).workloadLevel)); // state check
     }
 
+    @Override
+    public int compareTo(Workload other) {
+        return -workloadLevel.compareTo(other.getWorkloadLevel());
```

Comment:
Not sure why there's a minus sign here

Reply:
This is because default compareTo will make the task with lowest workload shown at the top of list. But I think the feature may be more useful to show the task with highest workload at first. How do you think about it?

Reply:
I see